components/StoryCards/HighlightedStoryCard.tsx

```diff
@@ -36,18 +36,16 @@ function HighlightedStoryCard({ story }: Props) {
                 </a>
             </Link>
             <div className={styles.content}>
-                <div className={styles.categoriesWrapper}>
-                    {/* 
+                {/* 
                         Since the position of date is changed the separator is no longer needed.
                         Commenting out instead of deleting in case we need to revert back.
```

Comment:
I think these commented blocks can be safely deleted :)

Reply:
removed!